js/src/product-feed/review-request/review-request-notice.test.js

```diff
@@ -44,13 +44,15 @@ describe( 'Request Review Notice', () => {
 
 		const { queryByText, queryByRole } = render(
 			<ReviewRequestNotice
-				account={ { status: 'BLOCKED', nextAttempt: 1644220897 } }
+				account={ { status: 'BLOCKED' } }
 				onRequestReviewClick={ onRequestReviewClick }
 			/>
 		);
 
 		expect(
-			queryByText( /You can request a new review on 07 Feb 2022/ )
+			queryByText(
+				/You can request a new review approximately 7 days after a disapproval./
+			)
```

Comment:
I agree that `toBeInTheDocument` is more descriptive. But since `queryByText` is returning `null`[ in case no matching elements](https://testing-library.com/docs/queries/about/#:~:text=queryBy...%3A%20Returns%20the%20matching%20node%20for%20a%20query%2C%20and%20return%20null%20if%20no%20elements%20match.%20This%20is%20useful%20for%20asserting%20an%20element%20that%20is%20not%20present.%20Throws%20an%20error%20if%20more%20than%20one%20match%20is%20found%20(use%20queryAllBy%20instead%20if%20this%20is%20OK).) and `isTruthy()` is all around our test suit I prefer to leave it like that except if it's creating actually a real issue.